components/clp-package-utils/clp_package_utils/controller.py

```diff
@@ -619,6 +620,30 @@ def _set_up_env_for_garbage_collector(self) -> EnvVarsDict:
 
         return env_vars
 
+    def _set_up_env_for_api_server(self) -> EnvVarsDict:
```

Comment:
sorry for not starting the discussion before you implement -

since both the MCP server and the garbage collector are optionally launched, we previously list them after the "always" launched services. (that - skipping the query services when Presto is enabled - is tricky but another story)

can we list the api-server before the webui? ditto in other files